l3kernel/l3int.dtx

```diff
@@ -1386,7 +1387,7 @@
       }
     #2 = \@@_eval:w #1 \@@_eval_end:
```

Comment:
Why the `\@@_eval:w #1 \@@_eval_end:` here? `#1` is already fully evaluated by `\int_eval:n` at this point, a `#1 \exp_stop_f:` should suffice.

Reply:
Checking back. that line goes a long way back. We used to directly define `\in_const:Nn`, so this was required, but split things out so we force evaluation of the input once here (to deal with random values, etc.). So we could simply here: but that's independent of the PR so I'll address separately.